omod/src/main/webapp/datasets/datasetEditor.jsp

```diff
@@ -135,6 +164,20 @@
 					</ul>
 				</td>
 			</tr>
+            <tr>
+                <td>
+		            <c:if test="${!empty dataSetDefinition.name}">
+			            <div>
```

Comment:
Just to help me out, could you checkout this indention locally, I have tried out other text editors but locally my indention is cool @dkayiwa

Reply:
Also with mac script editor, my indention is good @dkayiwa